CONTRIBUTING.md

```diff
@@ -0,0 +1,121 @@
+# Development Guide: Main Feast Repository
+> The higher level [Development Guide](https://docs.feast.dev/contributing/development-guide)
+> gives guidance on contributing to Feast codebase as a whole.
+
+### Overview
+This guide is targeted at developers looking to contribute to Feast components in
+the Main Feast Repository:
+- [Feast Python SDK / CLI](#feast-python-sdk-%2F-cli)
+- [Feast Documentation](#feast-documentation)
+- [Feast Go Client](#feast-go-client)
+- [Feast Terraform](#feast-terraform)
+
+> Don't see the Feast Component that you want to contribute to here?  
+> Check out the
+> [Development Guide](https://docs.feast.dev/contributing/development-guide)
+> to learn how Feast components are distributed over multiple repositories.
+
+
+## Feast Python SDK / CLI
+### Environment Setup
+Setting up your Development Environment for Feast Python SDK / CLI:
+1. Ensure you have `make`, Python (3.6 and above) with `pip`, installed.
+2. _Recommended:_ Create a virtual environment to isolate development dependencies to be installed
+```sh
+# create & activate a virtual environment
+python -v venv venv/
+source venv/bin/activate
+```
+
+3. Upgrade `pip` if outdated
+```sh
+pip install --upgrade pip
+```
+
+4. Install development dependencies for Feast Python SDK / CLI
+```sh
+pip install -r sdk/python/requirements-ci.txt
```

Comment:
This has changed. These files dont exist. Should be `pip install -e "sdk/python[ci]"` I believe. There is no more dev dependency any more either.

Reply:
Assuming that `pip install -e "sdk/python[ci]" is the be all and end all for dependencies for Python SDK developers.